cpython-unix/build.py

```diff
@@ -165,23 +165,7 @@ def add_target_env(env, build_platform, target_triple, build_env):
         # non-system (e.g. Homebrew) executables from being used.
         env["PATH"] = "/usr/bin:/bin"
 
-        if "APPLE_SDK_PATH" in os.environ:
-            sdk_path = os.environ["APPLE_SDK_PATH"]
-        else:
-            # macOS SDK has historically been in /usr courtesy of an
-            # installer provided by Xcode. But with Catalina, the files
-            # are now typically in
-            # /Applications/Xcode.app/Contents/Developer/Platforms/.
-            # The proper way to resolve this path is with xcrun, which
-            # will give us the headers that Xcode is configured to use.
-            res = subprocess.run(
-                ["xcrun", "--sdk", sdk_platform, "--show-sdk-path"],
-                check=True,
-                capture_output=True,
-                encoding="utf-8",
-            )
-
-            sdk_path = res.stdout.strip()
+        sdk_path = os.environ["APPLE_SDK_PATH"]
```

Comment:
This may break local builds outside of CI which don't have `APPLE_SDK_PATH` set.

Reply:
Ah ok -- I was partly trying to understand whether this is _always_ set in CI or if I was missing something. Makes sense.